app/src/main/java/fr/free/nrw/commons/edit/EditActivity.kt

```diff
@@ -66,13 +63,13 @@ class EditActivity : AppCompatActivity() {
                 ExifInterface.TAG_GPS_TIMESTAMP,
                 ExifInterface.TAG_IMAGE_LENGTH,
                 ExifInterface.TAG_IMAGE_WIDTH,
-                ExifInterface.TAG_ISO,
+                ExifInterface.TAG_PHOTOGRAPHIC_SENSITIVITY,
                 ExifInterface.TAG_MAKE,
                 ExifInterface.TAG_MODEL,
                 ExifInterface.TAG_ORIENTATION,
                 ExifInterface.TAG_WHITE_BALANCE,
-                ExifInterface.WHITEBALANCE_AUTO,
-                ExifInterface.WHITEBALANCE_MANUAL,
```

Comment:
For reviewers:
Deprecated in androidx
Ref : https://developer.android.com/reference/kotlin/androidx/exifinterface/media/ExifInterface#WHITEBALANCE_MANUAL()